package/MDAnalysis/core/topologyattrs.py

```diff
@@ -780,19 +780,42 @@ def get_segments(self, sg):
 
         return masses
 
-    def center_of_mass(group, pbc=None):
-        """Center of mass of the Group.
+    def center_of_mass(group, compound='group', pbc=None):
```

Comment:
`compound` is the right kwarg name to match [wrap](https://github.com/MDAnalysis/mdanalysis/blob/2e9db81a2ea2fb38b823cfbd33cdf37b9c898264/package/MDAnalysis/core/groups.py#L967)

Reply:
Is anything wrong with that?

Reply:
nope, this is correct, I'm mentioning because I had to look up what we'd previously called this, so I'm saving some other reviewers some time